inbc-program/inbc/parser/ota_parser.py

```diff
@@ -46,8 +46,9 @@ def sota(args: argparse.Namespace) -> str:
         'mode': args.mode,
         'release_date': release_date,
         'fetch': fetch_location,
+        'signature': args.signature,
         'username': args.username,
-        'password': _get_password(args.username, "Please provide the password: "),
+        'password': args.password if args.password else _get_password(args.username, "Please provide the password: "),
```

Comment:
passing password in via command line argument is a security issue on some systems--better to pass in through environment variable. (often any user can run `ps` and see arguments to running commands)

Reply:
The token is stored at /etc/intel_edge_node/tokens/release-service/rs_access_token in TiberOS. Either INBM or PUA can read from that file. INBM is the service that downloads the artifacts. I think we can simply ask INBM to read the password/token from that file without passing the password here and there.

Reply:
INBM will check the token from the file directly. This change has been reverted.